sdk/core/azure-core/azure/core/tracing/ext/opencensus_wrapper.py

```diff
@@ -111,24 +83,14 @@ def from_header(cls, headers):
         """
         Given a dictionary returns a new tracer with the span context extracted from that dictionary.
         :param headers: A key value pair dictionary
+        :type headers: Dict[str, str]
```

Comment:
mypy and rst have different syntax. I think this would be just `dict`. I usually render docs locally to have a look before publishing.

Reply:
done